src/components/Card/Card.css

```diff
@@ -134,7 +133,13 @@
   cursor: pointer;
 }
 
-.card .like-button .liked,
-.card .like-button:hover i {
+.card .like-button .liked, .card .like-button:hover i {
```

Comment:
I know it was like that but I would improve the performance of this css selectors.

I would say to change:
- From `.card .like-button .liked, .card .like-button:hover i` to `.card .liked, .card .like-button:hover i`

Reply:
Agreed - no need to add unnecessary CSS specificity. It can cause performance issues.

Reply:
Removed extra selector.